docs/apache-airflow-providers-amazon/operators/emr.rst

```diff
@@ -31,12 +31,12 @@ Overview
 
 Airflow to AWS EMR integration provides several operators to create and interact with EMR service.
 
- - :class:`~airflow.providers.amazon.aws.sensors.emr_job_flow.EmrJobFlowSensor`
- - :class:`~airflow.providers.amazon.aws.sensors.emr_step.EmrStepSensor`
- - :class:`~airflow.providers.amazon.aws.operators.emr_create_job_flow.EmrCreateJobFlowOperator`
- - :class:`~airflow.providers.amazon.aws.operators.emr_add_steps.EmrAddStepsOperator`
- - :class:`~airflow.providers.amazon.aws.operators.emr_modify_cluster.EmrModifyClusterOperator`
- - :class:`~airflow.providers.amazon.aws.operators.emr_terminate_job_flow.EmrTerminateJobFlowOperator`
+ - :class:`~airflow.providers.amazon.aws.sensors.emr.EmrJobFlowSensor`
+ - :class:`~airflow.providers.amazon.aws.sensors.emr.EmrStepSensor`
+ - :class:`~airflow.providers.amazon.aws.operators.emr.EmrCreateJobFlowOperator`
+ - :class:`~airflow.providers.amazon.aws.operators.emr.EmrAddStepsOperator`
+ - :class:`~airflow.providers.amazon.aws.operators.emr.EmrModifyClusterOperator`
+ - :class:`~airflow.providers.amazon.aws.operators.emr.EmrTerminateJobFlowOperator`
```

Comment:
This list is also missing `EmrContainerOperator` would you mind adding it in a followup PR?